pom.xml

```diff
@@ -107,7 +107,13 @@
                         <encoding>${project.build.sourceEncoding}</encoding>
                     </configuration>
                 </plugin>
+                <plugin>
+                    <groupId>org.apache.maven.plugins</groupId>
+                    <artifactId>maven-surefire-plugin</artifactId>
+                    <!-- JUnit 5 requires Surefire version 2.22.0 or higher -->
+                    <version>2.22.0</version>
+                </plugin>
             </plugins>
         </pluginManagement>
     </build>
-</project>
+</project>
```

Comment:
add empty line to the end of file

Reply:
she was
![Screenshot_3](https://user-images.githubusercontent.com/114933128/226304170-9476428e-2fc2-433a-b6c7-ba664be043c5.png)